pkg/webui/components/map/index.js

```diff
@@ -108,6 +108,7 @@ const MarkerRenderer = ({ marker }) => {
 
 const Controller = ({ onClick, centerOnMarkers, markers, bounds }) => {
   const map = useMap()
+  const [userInteracted, setUserInteracted] = useState(false)
```

Comment:
```suggestion
  const [useManualZoom, setUseManualZoom] = useState(false)
```

I think this is more descriptive. I was also thinking about what is the correct solution here, but ultimately I think your solution is how it should work. The map will automatically zoom based on the locations of the end devices, but once the user zooms in him/herself, this auto-zoom should be deactivated.